package.json

```diff
@@ -23,7 +23,7 @@
     "wiki:pull": "git subtree pull --prefix=.wiki https://github.com/SUI-Components/sui.wiki.git master --squash",
     "wiki:push": "git subtree push --prefix=.wiki https://github.com/SUI-Components/sui.wiki.git master",
     "commitmsg": "validate-commit-msg",
-    "precommit": "sui-precommit run"
+    "precommit": "sui-lint format-js --staged && sui-precommit run"
```

Comment:
This is just a proposal :-)
I think we should recommend to do this rather than making `sui-precommit run` to do it automatically...

Reply:
Why ?!

Reply:
Because it was adding files to the stage. But I'll change it.

Reply:
Yep!